### DISCUSSION TITLE
buddhism/2023/gayatri/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#109] [General]
# buddhism/2023/gayatri/

https://shanenull.com/buddhism/2023/gayatri/

**Comment by shane0** [comment]
https://en.wikipedia.org/wiki/Gayatri_Mantra

**Comment by shane0** [comment]
om - sacred syllable, primeval sound; bhur - physical body / physical realm;
bhuvah - life force / mental realm;
svah - soul / spiritual realm;
tat - that (God);
savitur - of Savitr (Sun, Creator, source of all life); varenyam - adore, venerable;
bhargo - splendour, effulgence, divine light;
devasya - Supreme Lord;
dhimahi - may we meditate;
dhiyo - buddhi (intellect);
yo - may this light;
nah - our;
prachodayāt - illumine / inspire.
